tcms/testplans/templates/testplans/get.html

```diff
@@ -89,7 +89,100 @@ <h2 class="card-pf-title">
+                                                <button type="button" class="btn btn-default dropdown-toggle"
+                                                        id="filter-button"
+                                                        data-toggle="dropdown"
+                                                        aria-haspopup="true"
+                                                        aria-expanded="false">{% trans 'Name' %}<span
+                                                        class="caret"></span></button>
+                                                <ul class="dropdown-menu">
+                                                    <li class="selected"><a href="#">{% trans 'Name' %}</a></li>
+                                                    <li><a href="#">{% trans 'ID' %}</a></li>
+                                                    <li><a href="#">{% trans 'Status' %}</a></li>
+                                                    <li><a href="#">{% trans 'Priority' %}</a></li>
+                                                </ul>
+                                            </div>
+                                            <button class="btn btn-link" type="button">
+                                                <span class="fa fa-sort-alpha-asc"></span>
+                                            </button>
+                                        </div>
+                                    </div>
+                                    <div class="form-group">
+                                        <button class="btn btn-default"
+                                                type="button">{% trans 'New TestRun' %}</button>
+                                        <button class="btn btn-default" type="button">{% trans 'Clone' %}</button>
+                                        <button class="btn btn-default"
+                                                type="button">{% trans 'Default tester' %}</button>
+                                        <div class="dropdown btn-group">
+                                            <button type="button" class="btn btn-default dropdown-toggle"
+                                                    id="filter-button"
+                                                    data-toggle="dropdown"
+                                                    aria-haspopup="true"
+                                                    aria-expanded="false">{% trans 'Status' %}<span
+                                                    class="caret"></span>
+                                            </button>
+                                            <ul class="dropdown-menu js-toolbar-status">
+                                                {% for status in statues %}
+                                                    <li><a data-id="{{ status.id }}">{{ status.name }}</a></li>
+                                                {% endfor %}
```

Comment:
Status & Priorities & all other buttons need adequate permissions check. See above in the deleted JavaScript code.

Reply:
I tried to disable the dropdown but I coulnd find a way to do so. But I think disabled the button will do the trick.